models/yang/sonic/import.mk

```diff
@@ -5,10 +5,15 @@
 # or glob patterns of basenames (like sonic-telemetry*.yang) can be specified.
 # Other sonic yangs referred by these will also be copied.
 #
+
+ifneq ($(SONIC_YANG_IMPORTS),)
+SONICYANG_IMPORTS = $(shell echo $(SONIC_YANG_IMPORTS))
```

Comment:
What values will be passed through SONIC_YANG_IMPORTS variable?? I don't see any usage.. Also, ` $(shell echo $(SONIC_YANG_IMPORTS))` will be same as `$(SONIC_YANG_IMPORTS)`